pretix_eth/migrations/0002_auto_20220529_2332.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.12 on 2022-05-29 23:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("pretixbase", "0208_auto_20220214_1632"),
+        ("pretix_eth", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="SignedMessage",
```

Comment:
SignedMessage is not necessary anymore. Artifact from the previous checkout implementation.

Can remove these migrations if desired.